src/quantile.rs

```diff
@@ -278,6 +300,27 @@ where
     S: Data<Elem = A>,
     D: Dimension,
 {
+    fn argmin(&self) -> Option<D::Pattern>
+    where
+        A: PartialOrd,
+    {
+        let min = self.first()?;
+        let mut pattern_min = self.dim();
```

Comment:
This is not correct do to my limited knowledge of `ndarray`, how do I get the pattern of the first element?